program-runtime/Cargo.toml

```diff
@@ -52,6 +51,9 @@ solana-transaction-context = { workspace = true }
 solana-type-overrides = { workspace = true }
 thiserror = { workspace = true }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
```

Comment:
You use `target_arch = "wasm32"` here, but `target_family = "wasm"` everywhere else. Can we use only `target_family`?

Reply:
@LucasSte Done. Would this change also go in the v2.1 branch?